### PR TITLE
PortfoliosSummaryResponse

### PR DESCRIPTION
**Pull request by Mejiao99**

### CODE REVIEW

File: pages/api/portfolios.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+export default function handler(req, res) {
+  res.status(200).json({
+    portfolios: [
```
Comment (jmena) [inline pages/api/portfolios.ts]: Delete this line.

Just return

```
res.status(200).json([
   {
     id: ...
     ...
}
```

File: pages/sec/index.tsx [hunk 2]
```diff
@@ -66,6 +66,14 @@ interface Portfolio {
   totalHoldings: Money;
 }
 
+interface PortfoliosSummaryResponse {
```
Comment (jmena) [inline pages/sec/index.tsx]: Delete this interface

File: pages/sec/index.tsx [hunk 3]
```diff
@@ -66,6 +66,14 @@ interface Portfolio {
...
+  portfolios: Portfolio[];
+}
+
+function getPortfoliosSummary() {
```
Comment (jmena) [inline pages/sec/index.tsx]: Delete this method.

File: pages/sec/index.tsx [hunk 4]
```diff
@@ -102,40 +110,20 @@ function CardPortfolios({ portfolios }) {
...
-      currency: "USD",
-    },
-  };
-
+export default function Index(props) {
   const [portfolios, setPortfolios] = useState([]);
   useEffect(() => {
     console.log("setPortfolios");
-    setPortfolios([port1, port2, port3]);
+    fetch(props.backendHost + "/hello")
```
Comment (jmena) [inline pages/sec/index.tsx]: Rename `/hello` to `portfolios`

File: public/static/portfolios.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+  "portfolios": [
```
Comment (jmena) [inline public/static/portfolios.json]: delete this line

File: pages/sec/index.tsx (hunk 3 above)
Comment (Mejiao99) [reply pages/sec/index.tsx]: resolve in next commit

File: pages/sec/index.tsx (hunk 2 above)
Comment (Mejiao99) [reply pages/sec/index.tsx]: resolve in next commit

File: pages/api/portfolios.ts (hunk 1 above)
Comment (Mejiao99) [reply pages/api/portfolios.ts]: resolve in next commit

File: public/static/portfolios.json (hunk 5 above)
Comment (Mejiao99) [reply public/static/portfolios.json]: resolve in next commit

File: pages/sec/index.tsx (hunk 4 above)
Comment (Mejiao99) [reply pages/sec/index.tsx]: resolve in next commit

**Pull request merged**